content/docs/user-guide/experiment-management/checkpoints.md

```diff
@@ -0,0 +1,454 @@
+dvclive.json  loss      0.04828  0.05095  0.00267
+dvclive.json  step      19       15       -4
+```
+
+_These are the same numbers you see in the metrics table, just in a different
+format._
+
+### Looking at plots
+
+You also have the option to generate plots to visualize the metrics about your
+training epochs. Running:
+
+```bash
+dvc plots diff 8fbab93 868c144
+```
+
+where `8fbab93` and `868c144` are the checkpoint ids you want to compare, will
+generate a `plots.html` file that you can open in a browser and it will display
+plots for you similar to the ones below.
+
+![Plots generated from running experiments on MNIST dataset using DVC](/img/checkpoints_plots.png)
+_The results here won't show anything interesting because the model increases in
+accuracy so fast._
+
+## Resetting checkpoints
+
+Usually when you start training a model, you won't begin with accuracy this
+high. There might be a time when you want to remove all of the existing
+checkpoints to start the training from scratch. You can reset your checkpoints
+with the following command:
+
+```bash
+dvc exp run --reset
+```
+
+This resets all of the existing checkpoints and re-runs the code to generate a
+new set of checkpoints under a new experiment branch.
+
+```dvc
+┏━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━┳━━━━━━┳━━━━━━━━━┳━━━━━━━━┳━━━━━━━━┳━━━━━━━━┳━━━━━━━━━━━━━━┓
+┃ Experiment            ┃ Created  ┃ step ┃    loss ┃    acc ┃ seed   ┃ lr     ┃ weight_decay ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━╇━━━━━━╇━━━━━━━━━╇━━━━━━━━╇━━━━━━━━╇━━━━━━━━╇━━━━━━━━━━━━━━┩
+│ workspace             │ -        │    4 │ 0.41666 │ 0.8777 │ 473987 │ 0.0001 │ 0            │
+│ main                  │ 04:49 PM │    - │       - │      - │ 473987 │ 0.001  │ 0            │
+│ │ ╓ exp-a75bb         │ 05:18 PM │    4 │ 0.41666 │ 0.8777 │ 473987 │ 0.0001 │ 0            │
+│ │ ╟ b8a4ceb           │ 05:18 PM │    3 │ 0.50835 │ 0.8538 │ 473987 │ 0.0001 │ 0            │
+│ │ ╟ 7b652b1           │ 05:17 PM │    2 │ 0.72421 │ 0.8284 │ 473987 │ 0.0001 │ 0            │
+│ │ ╟ cf0168f           │ 05:17 PM │    1 │  1.2537 │ 0.7738 │ 473987 │ 0.0001 │ 0            │
+│ ├─╨ 8c7f40f           │ 05:17 PM │    0 │  1.9428 │ 0.5715 │ 473987 │ 0.0001 │ 0            │
+│ │ ╓ exp-7fafd         │ 05:10 PM │    5 │ 0.36601 │ 0.8943 │ 473987 │ 0.0001 │ 0            │
+│ │ ╟ 9ac9e6a           │ 05:09 PM │    4 │ 0.41666 │ 0.8777 │ 473987 │ 0.0001 │ 0            │
+│ │ ╟ b869efe           │ 05:09 PM │    3 │ 0.50835 │ 0.8538 │ 473987 │ 0.0001 │ 0            │
+│ │ ╟ 422395e           │ 05:09 PM │    2 │ 0.72421 │ 0.8284 │ 473987 │ 0.0001 │ 0            │
+│ │ ╟ 44e44a9           │ 05:09 PM │    1 │  1.2537 │ 0.7738 │ 473987 │ 0.0001 │ 0            │
+│ ├─╨ c6e11b4           │ 05:09 PM │    0 │  1.9428 │ 0.5715 │ 473987 │ 0.0001 │ 0            │
+```
+
+We can also remove all of the experiments we don't promote to our Git workspace
+with the following command:
+
+```bash
+dvc exp gc --workspace --force
+```
```

Comment:
I'd suggest moving this to the bottom of the next section since it won't do anything if the users run it at this point before committing any experiment results to Git.